dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala

```diff
@@ -182,11 +182,11 @@ object EnceladusRestDAO extends EnceladusDAO {
 
         if (isUnauthorized) {
           log.warn(s"Unauthorized GET request for Menas URL: $url")
+          if (retriesLeft == 0) {
```

Comment:
would you mind putting `<= 0` in? Just o be safe if I don't know for what reason we decide in the future to manipulate the `restriesLeft`

Reply:
Maybe add `else log.info("trying to log in")`

Reply:
1. Sure.
2. Else is implied here and it does log it with a warning.